models/Location.v1.yaml

```diff
@@ -1,18 +1,25 @@
 title: Location
 description: "The root of the Location type's schema."
-required:
-  - id
-  - name
 type: object
+x-examples:
+  default:
+    id: 1
+    name: Default
+    max_capacity: 30
+    timezone_valid: true
+    timezone: 'Central Time (US & Canada)'
```

Comment:
The timezones should be following the TZ database maintained by the IANA (eg: `America/Vancouver`, `Europe/London`, etc)

Reply:
👍